setup.py

```diff
@@ -42,6 +42,7 @@ def read(file):
     'isort>=4.2.5',
     'collective.checkdocs>=0.2',
     'Pygments>=2.2.0',  # required by checkdocs
+    'mypy==0.501',
```

Comment:
It's a bit weird that we use non-pinned dependencies for all the other tools but fair enough, I get your point. :)